src/main/java/ticketservice/controller/TicketController.java

```diff
@@ -0,0 +1,22 @@
+package ticketservice.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import ticketservice.model.Ticket;
+import ticketservice.service.TicketService;
+
+
+@RestController
+public class TicketController {
+    private TicketService ticketService;
+
+    public TicketController(TicketService ticketService) {
+        this.ticketService = ticketService;
+    }
+
+    @GetMapping("/tickets/{id}")
+    Ticket getTicketById(@PathVariable int id) {
+        return ticketService.getTicket(id).orElseThrow(() -> new IllegalArgumentException("Can't find ticket with id: " + id));
```

Comment:
move logic about an error to service layer, because it will be displayed only in logs and is not related to MVC.
If you want to return specific error response to your client, read about `@ExceptionHandler` and `@ControllerAdvice`